src/content/1.7/themes/reference/javascript-events/_index.md

```diff
@@ -43,6 +43,27 @@ prestashop.emit(
 );
 ```
 
+### Listening to events
+
+You can also react to an event emitted by `prestashop.emit`. Here is a simple example:
+
+```js
+if (typeof prestashop !== 'undefined') {
+  prestashop.on(
+    'product updated',
```

Comment:
Line 88 below mentions `updatedProduct` event. Is that a mistake?

Else maybe we should keep both this part and the part after synchronized to help reader?

Reply:
Nice catch @matks !
I just update my PR, its more generic now !